src/tests/win-app-driver/FancyZonesTests.cs

```diff
@@ -44,6 +48,44 @@ public void EditorOpen()
             Assert.IsNotNull(editorWindow);
         }
 
+        [TestMethod]
+        /*
+         * click each toggle twice,
+         * save changes,
+         * check if settings are unchanged after clicking save button
+         */
+        [TestMethod]
```

Comment:
There are two `[TestMethod]`, and that is causing a build error.

Reply:
Sorry, it fixed

Reply:
Sorry, it fixed

Reply:
Thanks!